docs/components/stepper/index.html

```diff
@@ -49,7 +47,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Topic</span>
               <span class="beta-hxStepValue">Technical Incident</span>
```

Comment:
Please update all of these `beta` classes.

Reply:
Changes have been done, please review.